amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/AmqpChannelInitializer.java

```diff
@@ -28,14 +28,14 @@
     private final PulsarService pulsarService;
     @Getter
     private final AmqpServiceConfiguration amqpConfig;
-    private final AmqpTopicManager amqpTopicManager;
+    private final AmqpBrokerService amqpBrokerService;
 
     public AmqpChannelInitializer(PulsarService pulsarService, AmqpServiceConfiguration amqpConfig,
-                                  AmqpTopicManager amqpTopicManager) {
+                                  AmqpBrokerService amqpBrokerService) {
```

Comment:
Since the `AmqpBrokerService` already has an instance of the `PulsarService`, can it be removed from the function definition, and be pulled from the `amqpBrokerService.getPulsarService()`?

Reply:
Yes! good suggestion.